README.md

```diff
@@ -26,7 +26,7 @@ use dashcore_rpc::{Auth, Client, RpcApi};
 
 fn main() {
 
-    let rpc = Client::new("http://localhost:8332".to_string(),
+    let rpc = Client::new(&"http://127.0.0.1:19998".to_string(),
```

Comment:
Why the borrow? Also why switch out localhost?

Reply:
fixed - swaped localhost in for IP address